### ISSUE TITLE
Change name_triangle_spaced step to use visible character instead of spaces

### ISSUE THREAD

**Issue by alexmojaki** [alexmojaki/futurecoder#435]
A user inserted spaces before each character instead of after and couldn't see what was wrong.

**Comment by Yorwba** [comment]
I observed someone running into the same issue with this step. But wouldn't a more general solution be to highlight diffs between actual and expected output so that all kinds of small differences become easier to spot?

**Comment by alexmojaki** [comment]
Yes, that kind of thing would be great, it's just a lot more work to implement.
